sketch_map_tool/routes.py

```diff
@@ -220,6 +220,15 @@ def download(uuid: str, type_: REQUEST_TYPES) -> Response:
     return send_file(file, mimetype, download_name=download_name)
 
 
+@app.route("/api/ping")
+def ping():
+    """Ping Celery workers."""
+    result: list = celery_app.control.ping(timeout=1)
+    if result:
+        return {"status": "ok"}
+    return {"status": "fail"}
```

Comment:
I think this still returns a `2xx` status code. Is this on purpose?

Reply:
Its on purpose. I do not know of any good alternative. The endpoint did what it is supposed to do, check the celery worker and reports on it. 
I think it is fine like this.